js/obstaculos.js

```diff
@@ -0,0 +1,25 @@
+class Obstaculos{
+    constructor(ctx, canvasSize) {
+        this.ctx = ctx
+        this.obstacuSize = {
+            w: Math.random * 100,
```

Comment:
aqui me he dado cuenta que las medidas no tenain mucho sentido y le he dado valoras absolutos para comprobar si salian.
this.obstacuSize = {
            w: 100,
            h: 10
}
this.obstacuPos = {
            x: 250,
            y:0
        }
si metes esto en tu codigo vas a ver como te salen. Tu fallo esta en que el random no esta del todo correcto.